app/models.py

```diff
@@ -28,13 +105,16 @@ def __init__(self, title, location, time, date, description, cartegory, imageUrl
         self.description = description
         self.cartegory = cartegory
         self.imageUrl = imageUrl
+        self.created_by = created_by
 
     def save(self):
+        """Save an event to the database.
+        This includes creating a new event and editing one.
+        """
         db.session.add(self)
         db.session.commit()
 
     def add_rsvp(self, user):
```

Comment:
couldn't it be better to check if the user exists in an event's rsvplist instead of checking that this event exists in 'user.myrsvps?then in your models i don't see wherersvpslistandmyrsvps` are defined